roles/mesos/tasks/follower.yml

```diff
@@ -78,6 +78,15 @@
   tags:
     - mesos
 
+- name: set follower resources
+  sudo: yes
+  copy:
+    dest: /etc/mesos-slave/resources
+    content: "{{ mesos_resources }}"
+  notify: restart mesos follower
```

Comment:
I think I need to understand mesos follower metadata better. What is stored in there, and when is it ok to blow it away?